src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheckTest.java

```diff
@@ -131,6 +130,19 @@ public void testSingleLineLambda() throws Exception {
         verify(checkConfig, getPath("InputNeedBracesSingleLineLambda.java"), expected);
     }
 
+    @Test
+    public void testDoNotAllowSingleLineLambda() throws Exception {
+        final DefaultConfiguration checkConfig =
+            createModuleConfig(NeedBracesCheck.class);
+        checkConfig.addAttribute("tokens", "LAMBDA");
```

Comment:
please do me a favor and move this convig to block comment of InputNeedBracesSingleLineLambda class.
This input is used only in this Test.

Reply:
No, its not.
It is used in both `testSingleLineLambda` and `testDoNotAllowSingleLineLambda` tests, they have different configs (allow and forbid single line)

Reply:
Heh